ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -866,6 +924,25 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
+- as: azure-ipi-baselinecaps-v413-to-multiarch-p2-f14
```

Comment:
The workflow is `cucushift-installer-rehearse-azure-ipi-baselinecaps-v412`, so `v413` looks like a typo here

Reply:
ah, right. v413 has not supported yet. Thanks!

Reply:
yesterday, we just added v413 support, you can use v413 for testing now, but need to ensure the job title and workflow matched.

Reply:
Updated!